pkgs/by-name/ha/hanko/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, installShellFiles, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hanko";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "SRv6d";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-gytnUta/sNa8vJMvVS/DMwRc4R/8rmOnEfBKEjRpfGs=";
+  };
+  cargoHash = "sha256-rlKFS1ppUoyZ1X2PeG8tEB44Ysqi7kCLqR+ECVIdPjs=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installManPage assets/manpages/*.1
+
+    installShellCompletion assets/completions/hanko.bash
+    installShellCompletion assets/completions/_hanko
+    installShellCompletion assets/completions/hanko.fish
+  '';
+
+  meta = with lib; {
+    description = "Keeps your Git allowed signers file up to date with signing keys configured on software development platforms like GitHub and GitLab";
+    homepage = "https://github.com/SRv6d/hanko";
+    changelog = "https://github.com/SRv6d/hanko/blob/main/CHANGELOG.md";
+    license = licenses.mit;
+    maintainers = with maintainers; [ SRv6d ];
```

Comment:
The case used is wrong. It should be `[ srv6d ]` as written in the maintainers.nix file.

Reply:
Thanks, fixed.